Fixed #1495 bug : UseLoadTest missing claimsType

anjoy8 opened the pull request:
See details: https://github.com/dotnet-architecture/eShopOnContainers/issues/1495

sughosneo commented on src/Services/Basket/Basket.API/Infrastructure/Middlewares/ByPassAuthMiddleware.cs:
```diff
@@ -61,11 +61,12 @@ public async Task Invoke(HttpContext context)
+                    new Claim("http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name", "Test user"),
                     new Claim("nonce", Guid.NewGuid().ToString()),
                     new Claim("ttp://schemas.microsoft.com/identity/claims/identityprovider", "ByPassAuthMiddleware"),
                     new Claim("nonce", Guid.NewGuid().ToString()),
```
@anjoy8 , can we remove this duplicate claim ? Thanks.

dnfadmin commented:
[![CLA assistant check](https://cla.dotnetfoundation.org/pull/badge/not_signed)](https://cla.dotnetfoundation.org/dotnet-architecture/eShopOnContainers?pullRequest=1508) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet-architecture/eShopOnContainers?pullRequest=1508) before we can accept your contribution.<br/><br/>:x: anjoy8 [sign now](https://cla.dotnetfoundation.org/dotnet-architecture/eShopOnContainers?pullRequest=1508)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet-architecture/eShopOnContainers?pullRequest=1508) it.</sub>

anjoy8 commented:
For similar issues, I have updated other changes.

sughosneo commented on src/Services/Location/Locations.API/Infrastructure/Middlewares/ByPassAuthMiddleware.cs:
```diff
@@ -62,8 +62,7 @@ public async Task Invoke(HttpContext context)
-                    new Claim("ttp://schemas.microsoft.com/identity/claims/identityprovider", "ByPassAuthMiddleware"),
-                    new Claim("nonce", Guid.NewGuid().ToString()),
+                    new Claim("http://schemas.microsoft.com/identity/claims/identityprovider", "ByPassAuthMiddleware"),
                     new Claim("sub", "1234"),
```
Hey @anjoy8  - wouldn't be better to update the "sub" claim to `new Claim("sub", currentUserId)` for other `ByPassAuthMiddleware.cs` classes as well. Because I could see that `_identityService.GetUserIdentity()` is getting used for other scenarios as well ? For e.g `CreateOrUpdateUserLocationAsync` method in `LocationsController` class.

anjoy8 commented:
ok! 
I've modified the other files as well, 
and added username of ClaimType(http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name) to make things even more uniform.

sughosneo commented:
> ok!
> I've modified the other files as well,
> and added username of ClaimType(http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name) to make things even more uniform.

Looks good. Thank you. Will merge the changes.
